src/components/package/_fields/custom-coverage/package-coverage-fields.js

```diff
@@ -59,28 +75,65 @@ class PackageCoverageFields extends Component {
             type="text"
             component={Datepicker}
             label={<FormattedMessage id="ui-eholdings.date.endDate" />}
-            format={(value) => (value ? moment.utc(value) : '')}
+            format={formatField}
             validate={this.validateCoverageDate}
           />
         </div>
       </Fragment>
     );
   }
 
-  render() {
+  renderRepeatableField = (fieldArrayProps) => {
+    const {
+      fields,
+      name,
+    } = fieldArrayProps;
+
     const { initialValue } = this.props;
+    const { coverageDateAmount } = this.state;
+
+    const onAddField = () => {
+      fields.push({});
+
+      this.setState((prevState) => ({
+        coverageDateAmount: prevState.coverageDateAmount + 1,
+      }));
+    };
+
+    const onRemoveField = (index) => {
+      fields.remove(index);
+
+      this.setState((prevState) => ({
+        coverageDateAmount: prevState.coverageDateAmount - 1,
+      }));
+    };
+
+    const hasAddButton = coverageDateAmount === 0 || (coverageDateAmount === 1 && !initialValue[0]);
 
+    return (
+      <RepeatableField
+        addLabel={hasAddButton
```

Comment:
Please format this one as you did for empty message

```suggestion
        addLabel={
          hasAddButton
            ? <FormattedMessage id="ui-eholdings.package.coverage.addDateRange" />
            : null
        }
```